SPECS/bind/bind.spec

```diff
@@ -533,6 +533,9 @@ fi;
 
 %changelog
 * Mon Jun 21 2021 Thomas Crain <thcrain@microsoft.com> - 9.16.15-1
```

Comment:
The `1.0-dev` changelog entry for this version and release was done by Andrew. If you need to apply additional changes to this spec to make it work for `dev`, please keep the original version from `1.0-dev` and apply any fixes as a separate release.